Problem1/MethodProblem1.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SimpleProblem3
+{
+    public class MethodProblem1
+    {
+        public int SumOfEvenNumbers(int[] a)
+        {
+            int sum = 0;
+            for (int i = 0; i < a.Length; i++)
+            {
+                if (a[i] % 2 == 0)
+                    sum += a[i];
+            }
+            return sum;
+        }
```

Comment:
If you like working in C# you might want to look into LINQ. Here's a solution done using it:
```
public int SumOfEvenNumbers(int[] numbers) {
  return numbers.Where(number => number % 2 == 0).Aggregate(0, (result, number) => result + number);
}
```